.circleci/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:xenial
+
+RUN \
+  apt-get update &&\
+  apt-get install -y locales &&\
+  locale-gen en_US.UTF-8 &&\
+  update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+
+RUN \
+  apt-get install -y openjdk-8-jdk
+
+RUN \
+  echo "deb http://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list &&\
+  apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823 &&\
+  apt-get update &&\
+  apt-get install -y sbt
```

Comment:
I tried that, but I can couldn't find a suitable base image - I wanted ubuntu.

> how much difference it would make in the total build time

None at all. The built image is available from dockerhub and it's cached in circleci. At best we could have a slightly smaller Dockerfile.